### PR TITLE
Fixed meson r_version creation + reorder

### PR DESCRIPTION
**Pull request by xarkes**
Not sure if you'll appreciate the R2_VER thing in the Makefile, but why isn't it generated with acr ?

### CODE REVIEW

**Comment by XVilka** [comment]
@radare please check if it's OK for you? LGTM.

File: Makefile [hunk 1]
```diff
@@ -57,6 +57,7 @@ all: plugins.cfg libr/include/r_version.h
 #.PHONY: libr/include/r_version.h
 GIT_TAP=$(shell git describe --tags --match "[0-9]*" 2>/dev/null || echo $(VERSION))
 GIT_TIP=$(shell git rev-parse HEAD 2>/dev/null || echo HEAD)
+R2_VER=$(shell cat configure.acr | grep -w VERSION | head -n1 | awk '{print $$2}')
```
Comment (radare) [inline Makefile]: Remove the cat and use just grep

Comment (radare) [inline Makefile]: Why grep -w?

**Comment by radare** [comment]
Lgtm, after fixing my comments

**Pull request merged**